### ISSUE TITLE
Enter Cohort Detail to search all filed drop down option available for search the record 

### ISSUE THREAD

**Issue by zohaibihs** [OpenSRP/openmrs-module-cohort#40]
Enter Cohort Detail to search all filed drop down option available for search the record (UI Suggestion - normal)

**Comment by bholagabbar** [comment]
@zohaibihs  I'm sorry I did not understand this issue/comment. Could you elaborate. As far as dropdowns are concerned, I could only create those whose resources are available through REST so I would make AJAX calls again and again to refresh data

**Comment by zohaibihs** [comment]
I have analyzed that drop down option will not be a better option in case of large number of records, if possible all filed have Auto search option like "google Search" because new users have no information about Cohort name filed , Cohort program filed , Cohort head filed and Cohort Location to search for exact/particular  record(s).

![enter cohort detail to search](https://cloud.githubusercontent.com/assets/20788534/17398372/e722ca7a-5a55-11e6-9bee-bf5279e0ce2e.png)

**Comment by bholagabbar** [comment]
So basically you want me to implement a dynamic dropdown for every Field
correct?

**Comment by zohaibihs** [comment]
No, its not dynamic drop down  , its like "Google Search " Example - enter word  " Gmail" , So google auto Suggest word "gmail.com" .

![google search](https://cloud.githubusercontent.com/assets/20788534/17399953/9a5bb982-5a5e-11e6-8a9b-0ca521de34a2.png)

**Comment by bholagabbar** [comment]
Ummm I'm still not very clear as to what is expected... @maimoonak could
you help me out/tell me if this is feasible?

**Comment by maimoonak** [comment]
So its basiclly autopopulate for all fields. For Program you can use simple dropdown, but for others there is an autopopulate is requested. Though this is low priority and we can leave it for next iteration.
